joomla/joomla-cms
# [4.2] fix var name in content table

Fix for Pull Request #37017 .

### Summary of Changes

fix variable name: content table use `state` instead of `published`

### Testing Instructions

- try to create two articles with the same alias
- the existing article is either published or trashed
- check the error messages

### Actual result BEFORE applying this Pull Request

error message `COM_CONTENT_ERROR_UNIQUE_ALIAS_TRASHED="A trashed Article in this category has the same alias."` will not appear

![image](https://user-images.githubusercontent.com/66922325/199576268-7f99cf04-415f-4381-bcf1-7a75cc3154cd.png)


### Expected result AFTER applying this Pull Request

error message `COM_CONTENT_ERROR_UNIQUE_ALIAS_TRASHED="A trashed Article in this category has the same alias."` will appear

### Link to documentations
Please select:
